lib/logstash/inputs/kafka.rb

```diff
@@ -17,7 +17,7 @@
 # |0.9       |2.0.0 - 2.3.x   | 3.x.x |SSL |Works with the old Ruby Event API (`event['product']['price'] = 10`)  
 # |0.9       |2.4.x - 5.x.x   | 4.x.x |SSL |Works with the new getter/setter APIs (`event.set('[product][price]', 10)`)
 # |0.10.0.x  |2.4.x - 5.x.x   | 5.x.x |SSL |Not compatible with the <= 0.9 broker
-# |0.10.1.x  |2.4.x - 5.x.x   | 6.x.x |SSL |Not compatible with <= 0.10.0.x broker
+# |0.10.1.x  |2.4.x - 5.x.x   | 6.x.x |SSL, Kerberos SASL |Not compatible with <= 0.10.0.x broker
```

Comment:
The Kerberos SASL backported to 5.1.0 isn't in the change log. Was the backport done similarly in the output as well? If so, what plugin version?